meinberlin/assets/scss/components/_control-bar.scss

```diff
@@ -1,8 +1,18 @@
 .control-bar {
     @include clearfix;
 
+    &-search {
```

Comment:
i could not find an easy way to make the search input grow without using flexbox :(

`[searchbox ----->  grow                 ] [dropdown] [togglebutton]`